src/components/Nav/_nav-item.scss

```diff
@@ -16,7 +16,7 @@
   }
 
```

Comment:
Lots of nesting going on in this file... any room for refactoring the classnames to use modifiers? Can add to backlog if we want to get this merged asap.

Reply:
yeah it's a nasty file. I've added a TODO at the top so will create a backlog ticket